### DISCUSSION TITLE
Getting the body from a Got stream

### DISCUSSION THREAD

**Discussion by Richienb** [sindresorhus/got#1613] [Q&A] [answered]
#### What would you like to discuss?

After creating a `got` stream like so:

```js
const got = require("got")

const requestStream = got.stream("https://google.com")
```

I found that to get the result buffer this needs to be done:

```js
const pEvent = require("p-event")
const getStream = require("get-stream")

const [result] = await Promise.all([getStream.buffer(requestStream), pEvent(requestStream, "response")])
```

Is there a simpler way to do this?

#### Checklist

- [x] I have read the documentation.

**Comment by sindresorhus** [comment, marked as answer]
You should be able to just do `const data = await getStream(got.stream(…));`.

https://github.com/sindresorhus/got/blob/d0cd709fdd61d4e5762866309ede1b25cdbe8e60/test/stream.ts#L58

**Comment by Richienb** [comment]
👍🏻
